packages/jest-snapshot/src/index.ts

```diff
@@ -213,6 +231,48 @@ export const toMatchSnapshot: MatcherFunctionWithContext<
   });
 };
 
+export const toMatchNamedSnapshot: MatcherFunctionWithContext<
+  Context,
+  [snapshotName: string, properties?: object]
+> = function (received, snapshotName, properties?) {
+  const matcherName = 'toMatchNamedSnapshot';
+
+  if (properties !== undefined) {
+    if (
+      Array.isArray(properties) ||
+      typeof properties !== 'object' ||
+      properties === null
```

Comment:
The `toMatchSnapshot` logic got improved yesterday. I think this part should stay I sync:

```suggestion
      typeof properties !== 'object' ||
      properties === null
```

Reference: 

https://github.com/facebook/jest/blob/285b40d5954fd81e07b02db92185fd317ba92578/packages/jest-snapshot/src/index.ts#L166

Reply:
fixed